### ISSUE TITLE
Community Guidelines mis-formatted in mobile browsers / small browser windows

### ISSUE THREAD

**Issue by Alys** [HabitRPG/habitica#12525]
As reported by Mohammad Arsalan @hb-b5e2dq9vt15kmn6si (bef9b442-4f8b-4447-8cc4-91b2fb513b94):

"Community Guidelines page is not responsive for mobile view. Mostly at 375 * 667. Texts are not properly aligned."

Below are screenshots from the Firefox browser on my phone. The first one is with the page scrolled all the way to the top. Note how the mobile apps banner is hidden under the top nav bar. Ignore the white circle on the top right, which is from an unrelated overlay.

Note that when I view the Guidelines in the Habitica Android app, they're formatted better there, so fixing this isn't essential for mobile app users to see the Guidelines.

I'm labelling this as "needs design" for any design-related decisions to be made if desired. 

![image](https://user-images.githubusercontent.com/1495809/91840201-ae7d0d00-ec93-11ea-9579-eff6a01952b2.png)

![image](https://user-images.githubusercontent.com/1495809/91840478-192e4880-ec94-11ea-9737-4bf8a0fc8b0d.png)

**Comment by arsalanhub** [comment]
Yes. Not only at community guidelines page but also some places in user's dashboard for mobile view. Can I take this issue and work on it @Alys ??

**Comment by paglias** [comment]
@arsalanhub could you post a screenshot of the issue you're seeing?

**Comment by arsalanhub** [comment]
@paglias, screenshot is already posted & I have also reported possible px where this is happening.

**Comment by paglias** [comment]
@arsalanhub sorry I meant a screenshot of the issue happening in the user dashboard and not on the community guidelines page

**Comment by arsalanhub** [comment]
@paglias @alys As you can see level moved out of screen at 375 * 667


![habitica_dashboard](https://user-images.githubusercontent.com/53657473/92016568-29b3f100-ed70-11ea-9594-086e8a61e90c.png)

**Comment by citrusella** [comment]
Random comment that I felt the need to note: Most of the images in the community guidelines are for "flavor" but a few convey somewhat relevant information (but might out of date--i.e. the staff member image has more people than the people bulleted in the "from left to right" list, so it's unclear which avatar is which (and IIRC there might be staff not listed in the bullet list whose avatars are also not in the staff avatar image???)).

**Comment by paglias** [comment]
@citrusella did you mean to comment on this pr https://github.com/HabitRPG/habitica/pull/12500?

**Comment by citrusella** [comment]
No, I was mentioning it here because part of the issue (to me, anyway) is the images give the text less space, and the semi-important images are too small at that resolution to convey useful information.

It was more like a "thing to consider regarding formatting, just in case" kind of thing? o.o

**Comment by Nakonana** [comment]
Related issue: #12784 ?

**Comment by ChengCalvin** [comment]
Hi @Alys and @arsalanhub,

Is this issue still open? I am looking to contribute to my first open-source project, I might have encountered a similar issue building another website. Let me know!

**Comment by paglias** [comment]
@ChengCalvin this is still open indeed! I'm marking it as in progress for you. Let us know if you have any question

**Comment by ChengCalvin** [comment]
@paglias I am having errors when I try to run npm run client:dev (error: Cannot find module "@vue/cli-plugin-babel"). I checked in the dependencies and it is installed. Has anyone encountered this problem during the setup?

**Comment by paglias** [comment]
Not that I know @ChengCalvin . I would suggest to delete `node_modules` and `website/client/node_modules` and reinstall everything from the root of the project with `npm i`

**Comment by ChengCalvin** [comment]
Thanks, that did the trick. Is there a minimum memory requirement? I am getting another error when I run client:dev I get this error:
 
"FATAL ERROR: invalid array length Allocation failed - JavaScript heap out of memory"

![image](https://user-images.githubusercontent.com/22303125/100815230-71025a00-3411-11eb-85b2-6bb7a454b7fb.png)

**Comment by paglias** [comment]
I don't have an exact number but yeah the initial build of the client tends to take quite a bit of memory, how much RAM do you have available?

**Comment by ChengCalvin** [comment]
I don't have a lot of space left, I would need to buy new ram. I am currently still using 8GB total RAM which is not enough since I'm constantly above 75% usage.

**Comment by paglias** [comment]
Are you on windows, mac or linux? I just tried on mac and during the build phase it reaches 800-900mb of ram which then slowly decreases

**Comment by ChengCalvin** [comment]
I am on windows, when I try to build, it always stops around 68-69% before throwing that error. Seems like I don't have enough space, from my task manager it is showing 7.9GB in usage

**Comment by thewilloftheshadow** [comment]
I only hit 600mb when running on Ubuntu 20 LTS, so it might be a windows thing?

**Comment by paglias** [comment]
It definitely could be a windows only issue, but I'm afraid I won't be able to help as I haven't used windows in a long time

**Comment by ChengCalvin** [comment]
Yeah, I don't have enough RAM to continue after running npm run mongo:dev and npm run start I am left with <500mb to build. I will check if I am able to free up some space. Thanks

**Comment by thewilloftheshadow** [comment]
Try doing the initial build without running mongo or the server part, and with no other applications open, and see if that helps.

**Comment by ChengCalvin** [comment]
Seems like that made the build run a faster but the error persists.

**Update** 
I believe I found the problem in the report log, I don't have enough memory in my drive. 
available memory is less than the used, I will see where I can make space
![image](https://user-images.githubusercontent.com/22303125/101272190-f52d4800-3757-11eb-939c-ff08ccf42d53.png)

**Comment by seunggyupark** [comment]
Hey @ChengCalvin did you find a solution for this? I'm having trouble with this myself, same exact issue. I've seen this https://support.circleci.com/hc/en-us/articles/360009208393-How-can-I-increase-the-max-memory-for-Node- but unsure if this will help or how to apply that env variable.

**Comment by seunggyupark** [comment]
Also, interestingly, I also fail at 68-69% with 16gb of ram, windows. I don't think this is a hardware issue?

**Comment by ChengCalvin** [comment]
@seunggyupark can you check whether you have enough space in your C drive? I am still trying to find files to delete on it so I can't confirm yet. It seems that you need at least 11.3gb of memory available. You can check your report error log to see the memory available.

**Comment by seunggyupark** [comment]
I think memory it is referring to is RAM (random-access memory). I have 110gb free on my hard drive (C drive) and I only get up to about 4GB/16GB of RAM usage when compiling. 
![memory](https://user-images.githubusercontent.com/71467096/101549679-ee265580-3962-11eb-93fe-f7651ff9e5c7.PNG)

**Comment by ChengCalvin** [comment]
Yeah, I am in the same spot as you are. I was trying to verify my assumptions because available memory seemed to be what I had left in my drive and usedMemory I assumed it was what the build needed

**Comment by paglias** [comment]
I'm not familiar with windows env variables but looking on google this came out https://stackoverflow.com/questions/9249830/how-can-i-set-node-env-production-on-windows

**Comment by shanaqui** [comment]
Hi @ChengCalvin! Are you still working on this? If not, or if we don't hear back from you within a week, I'll mark it as help wanted again so the issue doesn't go totally stale. :)

**Comment by ChengCalvin** [comment]
Hey @shanaqui, unfortunately, I haven't been able to solve the issue with my computer. You can mark it as help wanted again. Thank you for letting me a chance at it!

**Comment by arsalanhub** [comment]
@Alys Can I take over the issue?

**Comment by anishlukk123** [comment]
can I work on this  @Alys  @paglias
